qiskit-superstaq/qiskit_superstaq/superstaq_backend.py

```diff
@@ -179,6 +181,7 @@ def compile(
     def _get_compile_request_json(
         self,
         circuits: Union[qiskit.QuantumCircuit, Sequence[qiskit.QuantumCircuit]],
+        dynamical_decoupling: bool = False,
```

Comment:
this shouldn't need to be added here, it'd be better to put this in the options dict